styles/components/_buttons.scss

```diff
@@ -325,6 +325,30 @@
   }
 }
 
+.btnFlx {
+  // see styleSheet.html for example usage.
+  // use for buttons in the new 'flex' style with more padding and use css flex.
+  @extend .btn;
+  padding: 25px;
+
+  border-radius: 0px;
+  border-top: 1px solid;
```

Comment:
This has to be border-top-width and border-top-style separately, or you're defaulting the color to black.